store.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma Store</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="Karma Logo" />
+      <nav>
+        <ul>
+          <li>
+            <a href="#"><b>Meet Karma</b></a>
+          </li>
+          <li>
+            <a href="#">How it Works</a>
+          </li>
+          <li>
+            <a href="#">Store</a>
+          </li>
+          <li>
+            <a href="#">Blog</a>
+          </li>
+          <li>
+            <a href="#">Help</a>
+          </li>
+          <li>
+            <a href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="form_image">
+      <section>
+        <div>
+          <h1>Order your Karma wifi device today!</h1>
+        </div>
+        <form action="">
+          <fieldset class="label_input">
+            <div>
+              <label for="first_name">First name *</label>
+              <input type="text" id="first_name" name="first_name" required />
+            </div>
+            <div>
+              <label for="last_name">Last name *</label>
+              <input type="text" id="last_name" name="last_name" required />
+            </div>
+          </fieldset>
+          <fieldset>
+            <label for="address">Address *</label>
+            <input type="text" id="address" name="address" required />
+          </fieldset>
+          <fieldset>
+            <label for="address2">Address 2</label>
+            <input type="text" id="address2" name="address2" />
+          </fieldset>
+          <fieldset class="label_input">
+            <div>
+              <label for="city">City *</label>
+              <select id="city" name="city" required>
+                <option disabled hidden selected>Select your city...</option>
+                <option>London</option>
```

Comment:
Each selectable option should have a value associated with it (this is so that the backend can know what was selected).